flocker/control/_persistence.py

```diff
@@ -202,7 +273,7 @@ def wire_encode(obj):
     :param obj: An object from the configuration model, e.g. ``Deployment``.
     :return bytes: Encoded object.
     """
-    return dumps(obj, cls=_ConfigurationEncoder)
+    return dumps(_cached_dfs_serialize(_cached_dfs_serialize(obj)))
```

Comment:
Wow yea that's not needed. Something something typo something something.

Thanks for catching that.